app/controllers/games_controller.rb

```diff
@@ -17,19 +17,25 @@ def show
     @game = Game.find(params[:id])
   end
 
+  def join
+    @game = Game.find(params[:id]).available
```

Comment:
In your join action, since you're using a POST action, Rails will be looking for a **:game_id** parameter in the URL address.  Let's change your game instance variable to this:

`@game = Game.find(params[:game_id])`

No need to use .available here because you're only concerned with the game object that is being joined and not all available games.